docs/team_batting_against.md

```diff
@@ -0,0 +1,23 @@
+# Team Batting Against Stats from Baseball-Reference
+
+`team_batting_against(season=[most_recent_season])`
```

Comment:
super nitpick - `[most_recent_season]` makes it look like this should be a list. Maybe drop the brackets